huntsman/tests/test_aardvark.py

```diff
@@ -33,7 +37,49 @@ def test_name_server(name_server):
 def test_locate_name_server(name_server):
     # Check that we can connect to the name server
     Pyro4.locateNS()
-    
+
+
+def test_test_server(test_server):
+    # Check that it is running
+    assert test_server.poll() is None
+
+
+def test_quantity_argument(test_proxy):
+    test_proxy.quantity_argument(550 * u.nm)
+    test_proxy.quantity_argument(1.21 * u.GW)
+
+
+def test_quantity_return(test_proxy):
+    q = test_proxy.quantity_return()
+    assert isinstance(q, u.Quantity)
+    assert q == 42 * u.km * u.ng / u.Ms
```

Comment:
I don't understand why `quantity_return` with no params would be equal to `42 * u.km * u.ng / u.Ms`

Reply:
I see below what this is doing. I guess since this is all internal to testing it's not terribly important.